PERMISSION_FIX.md

```
@@ -0,0 +1,293 @@
+WHERE r.name = 'USER' AND p.name IN ('COURSE_READ');
+```
+
+#### Option C: Đơn giản hóa - Không dùng @PreAuthorize
+
+**Trong CourseController.java:**
+```java
+// Bỏ @PreAuthorize, chỉ cần authenticated()
+@GetMapping
+// @PreAuthorize("hasAuthority('COURSE_READ')") // Bỏ dòng này
+public ResponseEntity<ApiResponse<Page<CourseResponse>>> getAllCourses(...)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)